web/src/page_params.ts

```diff
@@ -1,5 +1,39 @@
 import $ from "jquery";
 
+export const all_repository_names = [
+    "zulip",
+    "zulip-desktop",
+    "zulip-mobile",
+    "python-zulip-api",
+    "zulip-js",
+    "zulipbot",
+    "zulip-terminal",
+    "zulip-ios-legacy",
+    "zulip-android",
+] as const;
+type RepositoryName = (typeof all_repository_names)[number];
+
+export const repo_name_to_tab_name: Record<RepositoryName, string> = {
+    zulip: "server",
+    "zulip-desktop": "desktop",
+    "zulip-mobile": "mobile",
+    "python-zulip-api": "python-zulip-api",
+    "zulip-js": "zulip-js",
+    zulipbot: "zulipbot",
+    "zulip-terminal": "terminal",
+    "zulip-ios-legacy": "",
+    "zulip-android": "",
+};
+
+export type Contributor = {
+    avatar: string;
+    email?: string;
+    github_username?: string;
+    name: string;
+} & {
+    [K in RepositoryName]?: number;
+};
```

Comment:
This change is wrong -- we need a `web/src/portico/team_page_params.ts`. The page parameters sent for this other page are unrelated to those for the main web app.

Here are some examples:
```
$ ls web/src/*/*param*
web/src/billing/page_params.ts  web/src/stats/page_params.ts
```

Reply:
`src/page_params` also runs for portico pages. So, I extracted a `team_page_params_types` as next best thing I could think of.